### PR TITLE
persisted scores on switching users

### PR DESCRIPTION
**Pull request by yungalyx**
@dchen150 The new data coming from firebase may have broken your sorting function a little bit

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import Router from './components/router'
+import Router from './components/Router'
```
Comment (dchen150) [inline src/App.js:2]: can we leave it lowercase?

File: src/components/Router.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 import Landing from '../pages/landing'
-import Assessments from '../pages/assessments'
+import Assessments from '../pages/Assessments'
```
Comment (dchen150) [inline src/components/Router.js:4]: lowercase?

File: src/components/applicantScore.js [hunk 3]
```diff
@@ -2,36 +2,101 @@
...
+                PassionScore: null,
+            })
+            setHasScore(false)
+        }
+    }, [props])
+
+
+    const handleClick = async (value, label) =>{
+        switch(label){
+            case "Github/Personal Website":
```
Comment (dchen150) [inline src/components/applicantScore.js:55]: can we make the label values constants so we can change them easily if needed

File: src/components/applicantScore.js [hunk 4]
```diff
@@ -2,36 +2,101 @@
...
+                await updateApplicantScore('LHD2021', props.hacker.firebaseID, {"score.MainInterestScore": value})
+                break
+            case "Tell us a recent project you've worked on that you're proud of!":
+                props.passingUpScore(props.hacker.firebaseID, {...score, PassionScore:value})
+                setScore({...score, PassionScore: value})
+                await updateApplicantScore('LHD2021', props.hacker.firebaseID, {"score.PassionScore": value})
+                break
+            default:
+                alert('Error!')
+                break
```
Comment (dchen150) [inline src/components/applicantScore.js:77]: don't think break is needed for default case

File: src/pages/Assessments.js [hunk 5]
```diff
@@ -1,93 +1,44 @@
 import React from 'react'
 import { useState, useEffect } from 'react'
 import testPDf from '../assets/testResume.pdf'
-import Table from '../components/table'
+import Table from '../components/Table'
```
Comment (dchen150) [inline src/pages/Assessments.js:4]: lowercase

File: src/pages/Assessments.js [hunk 6]
```diff
@@ -1,93 +1,44 @@
...
+    // trying to move the userID (key) into values side
+      
+    for (const key in data) {
+      data[key].firebaseID = key
+    }
     const arr = Object.values(data)
     setHackers(arr)
     setDisplayedHackers(arr)
-    console.log(arr)
+    // console.log(arr)
```
Comment (dchen150) [inline src/pages/Assessments.js:40]: remove

**Review by dchen150 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (yungalyx) [reply src/App.js:2]: my vscode won't let me run it if i have it lowercase

Comment (dchen150) [reply src/App.js:2]: what error does it show?

**Comment by Ciuffi** [comment]
I made some changes that rely on this branch so i'm force merging

**Pull request merged**